enhancements/microshift/dummy-gateway-interface-ovn.md

```diff
@@ -0,0 +1,477 @@
+  │    │     │        ┌────────┴────────┐                   │    │
+  │    │     │        │ External Switch │                   │    │
+  │    │     │        └────────┬────────┘                   │    │
+  │    │     │                 │                            │    │
+  │    │     │     ┌───────────┴──────────┐                 │    │
+  │    │     │     │     Gateway Router   │                 │    │
+  │    │     │     └───────────┬──────────┘                 │    │
+  │    │     │                 │                            │    │
+  │    │     │         ┌───────┴────────┐                   │    │
+  │    │     │         │  Join Switch   │                   │    │
+  │    │     │         └───────┬────────┘                   │    │
+  │    │     │                 │                            │    │
+  │    │     │   ┌─────────────┴─────────────┐              │    │
+  │    │     │   │    OVN Cluster Router     │              │    │
+  │    │     │   │        10.244.0.1/24      │              │    │
+  │    │     │   └─────────────┬─────────────┘              │    │
+  │    │     │                 │                            │    │
+  │    │     │        ┌────────┴─────────┐                  │    │
+  │    │     │        │    Node Switch   │                  │    │
+  │    │     │        └────────┬─────────┘                  │    │
+  │    │     │                 │                            │    │
+  │    │  ┌──┴────────────┐    │      ┌──────────────┐      │    │
+  │    │  │ mp0           ├────┴──────┤Pod           │      │    │
+  │    │  │ 10.244.0.2/24 │           │10.244.0.50/24│      │    │
+  │    │  └───────────────┘           └──────────────┘      │    │
+  │    │                                                    │    │
+  │    └────────────────────────────────────────────────────┘    │
+  │                                                              │
+  └──────────────────────────────────────────────────────────────┘
+```
+
+There will be no change to the overlay path taken by the east-west traffic
+between the pods.
+
+The following north-south traffic paths will not be affected either:
+
+* Overlay pod to external
+  * This traffic path is the same as normal local gw mode since the egress
+    traffic will leave OVN-K virtual topology via mp0. Then it will be
+    masqueraded by the host iptables rule and then be sent out from eth0.
+* Host to ClusterIP services backed by overlay pods
+* Host to NodePort services backed by overlay pods
+
+Since we will use a static IP address (10.44.0.0) on br-ex other than the NodeIP
+(10.89.0.3) which is on eth0. OVN-K will treat the NodeIP as an external IP. The
+endpoint IP of service backed by hostNetwork pod will still use NodeIP. Also the
+ingress traffic to NodePort Service will still use the NodeIP as the dest_ip.
+Therefore the following traffic cases will be affected:
+
+* Host to ClusterIP services backed by host pods
```

Comment:
call our service of type `LoadBalancer` backed by host pods ?

Reply:
The traffic to LB service is DNATed to clusterIP by iptables rule first. This is almost the same as nodePort backed by host pods.